data_quality/main.py

```diff
@@ -94,16 +76,17 @@ def generate(generator_class, endpoint, config_filepath, generator_path, file_ty
 
         generator_class: Name of the generator class (ex: CkanGenerator)
         endpoint: Url where the generator should get the data from
-        config_filepath: Path to the json config for data-quality-cli
     """
     if generator_class not in generators._built_in_generators:
         if generator_path is None:
             raise click.BadParameter(('You need to provide the path for your custom'
                                      'generator using the `--generator_path` option.'))
 
     file_types = list(file_type)
-    config = helpers.load_json_config(config_filepath)
-    config['data_dir'] = helpers.resolve_relative_path(config_filepath, config['data_dir'])
+    config = utilities.load_json_config(config_filepath)
+    if not config_filepath:
+        config['data_dir'] = utilities.resolve_relative_path(os.getcwdu(), config['data_dir'])
```

Comment:
The reason why `config_filepath` was mandatory before was mainly that it provided a good way to resolve the `data_dir` full path if the one provided in the config was relative. If this default config is used, the default path is the one where the command is executed (at least on Unix). Opinion on these needed.